iOS6-HW26-Alexander.Strelkov/Sources/Scenes/DetailView/UserDetailsViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  UserDetailsViewController.swift
+//  iOS6-HW26-Alexander.Strelkov
+//
+//  Created by Alexandr Strelkov on 20.08.2022.
+//
+
+import UIKit
+import CoreData
+
+class UserDetailsViewController: UIViewController {
+    
+    //MARK: -Attributes:
+    
+    var isEdit = false
+    var user = UserDataManager.instance.user
+    
+    //MARK: -IBOutles:
+    
+    @IBOutlet weak var nameTextField: UITextField!
+    
+    @IBOutlet weak var dateTextField: UITextField!
+    
+    @IBOutlet weak var genderTextField: UITextField!
+    
+    @IBOutlet weak var editButton: UIBarButtonItem!
```

Comment:
Это можно и слепить вместе, на читаемость никак не влияет))
```suggestion
    @IBOutlet weak var nameTextField: UITextField!
    @IBOutlet weak var dateTextField: UITextField!
    @IBOutlet weak var genderTextField: UITextField!
    @IBOutlet weak var editButton: UIBarButtonItem!
```